instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_client.py

```diff
@@ -122,6 +123,9 @@ def _start_guarded_span(self, *args, **kwargs):
         return _GuardedSpan(self._start_span(*args, **kwargs))
 
     def intercept_unary(self, request, metadata, client_info, invoker):
+        if context.get_value(_SUPPRESS_INSTRUMENTATION_KEY):
```

Comment:
Does this functionality need to be added to the server interceptor too?

Reply:
From all the implementations under contrib repo,
`_SUPPRESS_INSTRUMENTATION_KEY` is mainly used under the client side instead of server side.
So, I did not implement the server side at first place.
My preference is first to make sure the client side works as expected.
Then, we can investigate the server side implementation later on.
I want to make each PR only containing a single feature.

Reply:
It should definitely be done for the server side as well if you get a chance 😃

Reply:
@RyanSiu1995 
Would you like to contribute that in a separate PR as well?

Reply:
Yes, I will!